catalog/spatial/geocoding/spatial-geocoding-offline-catalog/src/main/java/org/codice/ddf/spatial/geocoding/index/GeoNamesCatalogIndexer.java

```diff
@@ -142,15 +142,6 @@ private Metacard transformGeoEntryToMetacard(GeoEntry geoEntry) {
     metacard.setAttribute(
         new AttributeImpl(
             GeoEntryAttributes.FEATURE_CODE_ATTRIBUTE_NAME, geoEntry.getFeatureCode()));
-    Integer gazetteerSortValue = getGeoNameGazetterSortByFeatureClass(geoEntry);
```

Comment:
Why was this removed? Not sure what it is trying to accomplish but seems unrelated to the overall goal of the pr.

Reply:
It was from a merge conflict. I'll revert that.